### ISSUE TITLE
Cura Build Freezes on `test_sockets.py`

### ISSUE THREAD

**Issue by JonathanILevi** [NixOS/nixpkgs#91544]
**Describe the bug**
When installing Cura with `nix-env` it freezes while, apparently, running tests.
Here is the command-line output of nix-env: https://gist.github.com/JonathanILevi/a87892e2d80ce4b14d18dbc3898f93e1

**To Reproduce**
Run `nix-env -iA unstable-pkgs.cura`.
(My OS is using `nixos-stable` channel, but I am installing Cura with `nixos-unstable`).

**Notify maintainers**
@gebner @OmnipotentEntity @nh2 @esclear @betaboon

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.43, NixOS, 20.03.2015.e7752db2fb6 (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.5`
 - channels(root): `"nixos-20.03.2351.f8248ab6d9e, spkgs-20.03.2351.f8248ab6d9e, upkgs-20.09pre231796.22a81aa5fc1"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by OmnipotentEntity** [comment]
With the current HEAD of master, I cannot reproduce this issue.  However, I do get a different error during the tests of `sentry-sdk-0.14.4` (which occurs far beyond the point that you experienced failures in your build.)

My error is:

```
running egg_info
writing sentry_sdk.egg-info/PKG-INFO
writing dependency_links to sentry_sdk.egg-info/dependency_links.txt
writing requirements to sentry_sdk.egg-info/requires.txt
writing top-level names to sentry_sdk.egg-info/top_level.txt
reading manifest file 'sentry_sdk.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'sentry_sdk.egg-info/SOURCES.txt'
running build_ext
trytond (unittest.loader._FailedTest) ... ERROR

======================================================================
ERROR: trytond (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: trytond
Traceback (most recent call last):
  File "/nix/store/f87w21b91cws0wbsvyfn5vnlyv491czi-python3-3.8.3/lib/python3.8/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "/build/sentry-sdk-0.14.4/sentry_sdk/integrations/trytond.py", line 7, in <module>
    from trytond.exceptions import TrytonException  # type: ignore
ModuleNotFoundError: No module named 'trytond'


----------------------------------------------------------------------
Ran 1 test in 0.001s

FAILED (errors=1)
Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
error: Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
builder for '/nix/store/8lbaa3nmxp12dv1iw1wa2549zvbm7qwr-python3.8-sentry-sdk-0.14.4.drv' failed with exit code 1
cannot build derivation '/nix/store/hyxm1pgl7mpqfz82xwhawf59fnn6vjah-cura-4.6.1.drv': 1 dependencies couldn't be built
error: build of '/nix/store/hyxm1pgl7mpqfz82xwhawf59fnn6vjah-cura-4.6.1.drv' failed
```

It looks like the pip package [`trytond`](https://pypi.org/project/trytond/) is not in nixpkgs, hence the failure.  Probably should skip that test, or package `trytond`.

**Comment by OmnipotentEntity** [comment]
Running against the nixos-unstable branch as indicated by your initial post; however, I get the exact same error.  

It's likely caused by the build issue fixed in #91440 yesterday.  So if you wait until unstable is synced up with master again you'll be able to get my error instead of yours.

**Comment by gebner** [comment]
The sentry-sdk failure is being fixed in #91417.

**Comment by gebner** [comment]
Fixed by #91417
